ReactiveCocoaTests/Swift/SignalProducerLiftingSpec.swift

```diff
@@ -496,6 +496,114 @@ class SignalProducerLiftingSpec: QuickSpec {
 				observer.sendFailed(.Default)
 				expect(error) == TestError.Default
 			}
+
+			it("should collect a certain count of values") {
+				let (original, observer) = SignalProducer<Int, NoError>.buffer(1)
+
+				let producer = original.collect(3)
+
+				var expectedValues = [
+					[1, 2, 3],
+					[4, 5, 6],
+					[7, 8, 9],
+					[0]
+				]
```

Comment:
As written, this test doesn't make any guarantees about the timing of the values—whether this was written as `inclusive` or `exclusive`. Can you rewrite the test to match the style of many of the other RAC tests, where the timing of each value can be verified?